Buildings/Controls/OBC/ASHRAE/PrimarySystem/BoilerPlant/package.mo

```diff
@@ -0,0 +1,5 @@
+within Buildings.Controls.OBC.ASHRAE.PrimarySystem;
```

Comment:
Missing annotation with the `Documentation` and `Icon` - as an example please see how the `ChillerPlant` package.mo looks like with a text editor.